pkg/codegen/testing/test/testdata/output-funcs/go/mypkg/funcWithEmptyOutputs.go

```diff
@@ -4,66 +4,17 @@
 package mypkg
 
 import (
-	"context"
-	"reflect"
-
 	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
 )
 
 // n/a
-func FuncWithEmptyOutputs(ctx *pulumi.Context, args *FuncWithEmptyOutputsArgs, opts ...pulumi.InvokeOption) (*FuncWithEmptyOutputsResult, error) {
-	var rv FuncWithEmptyOutputsResult
+func FuncWithEmptyOutputs(ctx *pulumi.Context, args *FuncWithEmptyOutputsArgs, opts ...pulumi.InvokeOption) error {
```

Comment:
This is scary, since it ensures that every time a function with no outputs is called, this always breaks the call site. Do we have any idea how many invokes we have that don't return any outputs?

Reply:
> This is scary, since it ensures that every time a function with no outputs is called, this always breaks the call site.

Note that this was already the case when `*Function.Outputs == nil` I only extended this to be:
```
Outputs == nil || len(Outputs.Properties) > 0
```
so that no outputs and empty outputs are the same thing

> Do we have any idea how many invokes we have that don't return any outputs?

I don't know tbh but it is something we already supported so here we are not introducing a new concept